src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DictionaryAnalysisData.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.CodeAnalysis.FlowAnalysis.DataFlow
     public sealed class DictionaryAnalysisData<TKey, TValue> : AbstractAnalysisData, IDictionary<TKey, TValue>
     {
 #pragma warning disable CA2213 // Disposable fields should be disposed
-        private PooledDictionary<TKey, TValue> _coreAnalysisData;
+        private PooledDictionary<TKey, TValue>? _coreAnalysisData;
```

Comment:
💡 I recommend _not_ changing this field, and instead changing the null assignment to use `null!`. Most of the rest of the changes in the file can be reverted at that point.